### PR TITLE
README typo, build profile addition and version bump

### PR DESCRIPTION
**Pull request by randerzander**
Added interpreter/ folder to .gitignore

README.md
Fixed a typo in README.md
Updated mvn build command for Hadoop 2.7

In spark/pom.xml:
Bumped Spark default version number from 1.4.0 to 1.4.1
Added -PHadoop-2.7 build profile

### CODE REVIEW

**Comment by felixcheung** [comment]
+1

File: spark/pom.xml [hunk 1]
```diff
@@ -668,6 +668,16 @@
     </profile>
 
     <profile>
+      <id>hadoop-2.7</id>
```
Comment (Leemoonsoo) [inline spark/pom.xml]: About adding profile for hadoop-2.7, it was discussed in #158 and decided not to have separate profile.
I'm okay with both changing / keeping decision. I'd like to ask @randerzander @jongyoul @fantazic @felixcheung opinions.

Comment (randerzander) [inline spark/pom.xml]: I have no opinion on whether a new Hadoop profile is NEEDED. I followed the trend of new profile per Hadoop release. I'm fine using -Phadoop-2.6, but then we should add a line in the README indicating Hadoop 2.7 users should continue using the -Phadoop-2.6 build option.

Comment (felixcheung) [inline spark/pom.xml]: I see that Hortonworks HDP 2.3 released recently (or soon) has Hadoop 2.7.1, so maybe there is a reason to have that profile.


With so many profiles I can see it is increasing difficult to cover all of it in each release. Do we have any plan to trim the older ones?



On Sat, Jul 25, 2015 at 3:19 PM -0700, "Randy Gelhausen" <notifications@github.com> wrote:
> @@ -668,6 +668,16 @@
>      </profile>
>
>      <profile>
> +      <id>hadoop-2.7</id>

I have no opinion on whether a new Hadoop profile is NEEDED. I followed the trend of new profile per Hadoop release. I'm fine using -Phadoop-2.6, but then we should add a line in the README indicating Hadoop 2.7 users should continue using the -Phadoop-2.6 build option.

---
Reply to this email directly or view it on GitHub:
https://github.com/apache/incubator-zeppelin/pull/166/files#r35487366

File: README.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ mvn clean package -Pspark-1.1 -Dhadoop.version=2.6.0 -Phadoop-2.6 -Pyarn -DskipT
 ```
 Yarn (Hadoop 2.7.x)
 ```
-mvn clean package -Pspark-1.1 -Dhadoop.version=2.7.0 -Phadoop-2.6 -Pyarn -DskipTests
```
Comment (jongyoul) [inline README.md]: `-Phadoop-2.6` is enough to support hadoop 2.7.x for now like using `-Phadoop-2.4` for building it with hadoop 2.5

**Comment by Leemoonsoo** [comment]
Looks good to me!

**Comment by jongyoul** [comment]
LGTM!

**Pull request closed without merging**